wfl/Database/MongoDBManager.cs

```diff
@@ -19,69 +20,60 @@ public MongoDBManager()
             db = client.GetDatabase("WFL");
         }
 
-        public async Task<Restaurant> InsertRestaurantAsync(RestaurantDto r)
+        private string GenerateCollectionName<T>()
```

Comment:
I'll add another issue for this change so that it can be addressed in its own PR.